Sustainsys.Saml2/SAML2P/Saml2Response.cs

```diff
@@ -602,7 +602,7 @@ private IEnumerable<ClaimsIdentity> CreateClaims(IOptions options, IdentityProvi
             }
 
             TokenValidationParameters validationParameters = options.SPOptions.TokenValidationParametersTemplate.Clone();
-            validationParameters.ValidAudience = options.SPOptions.EntityId.Id;
+            validationParameters.ValidAudience = idp.spOptions.EntityId.Id;
```

Comment:
Just checking that this is intentional

Reply:
no, it was not, did not realize that I changed so many files